### ISSUE TITLE
KeyPass as autofill service

### ISSUE THREAD

**Issue by J4k3v3** [yogeshpaliyal/KeyPass#831]
**Feature:**
I wanna see KeyPass as android autofill service. Also, I want to see adding and saving credit card info in this app, like in "KeyGo" app That's a so useful features for all of us. Can you implement this ideas?

**Comment by fallsdevil** [comment]
I agree, I'd just make that suggestion, the app is very good and I wanna to quit Bitwarden because the auto-fill doesn't work without opening the app and that's annoying.

**Comment by ippocratis** [comment]
Whats the use of a password manager on android without auto fill?

I like this app much and want to switch from bitwarden but without auto fill its a no-go.

**Comment by yogeshpaliyal** [comment]
https://github.com/yogeshpaliyal/KeyPass/issues/342
